external/lfric_infrastructure/preprocessed/Makefile

```diff
@@ -0,0 +1,16 @@
+# This Makefile is automatically created by ../update.sh
+# ======================================================
+
+# Don't modify the file directly, if required, update ../update.sh
```

Comment:
There's probably an obvious answer to this but, if this file is generated, why is it under revision control?

Reply:
See comment earlier: based on the assumption that a user can run CI tests without having to trigger preprocessing first. I've updated the comment a bit, maybe that helps?